src/components/calcite-slider/calcite-slider.tsx

```diff
@@ -211,6 +622,89 @@ export class CalciteSlider {
       </div>
     ) : null;
   }
+
+  private renderTickLabel(tick: number): VNode {
+    const isMinTickLabel = tick === this.min;
+    const isMaxTickLabel = tick === this.max;
+    const tickLabel = (
+      <span
+        class={{
+          tick__label: true,
+          "tick__label--min": isMinTickLabel,
+          "tick__label--max": isMaxTickLabel,
+        }}
+      >
+        {tick.toLocaleString()}
+      </span>
+    );
+    if (this.labelTicks && !this.hasHistogram && !this.isRange) {
+      return tickLabel;
+    }
+    if (
```

Comment:
all of these if's return the same thing?

Reply:
Correct, this is another example of what made it easier for me in development of this portion to literally list out each specific prop combination as separate if statements.  I could probably consolidate these into a big if with `||` in between.  That would certainly cut down the visible lines of code.

Reply:
The more I think about this, I feel like combining all of these myriad combinations together into a single `if` statement will make it even messier and harder to read.  Prettier is already breaking each condition onto its own line, so to combine these all into one `if` would make probably the biggest `if` condition in javascript history, lol.